src/factories/OrchestratorFactory.sol

```diff
@@ -1,5 +1,8 @@
 // SPDX-License-Identifier: LGPL-3.0-only
-pragma solidity 0.8.19;
+pragma solidity 0.8.20;
```

Comment:
Versions needs to be updated to `0.8.23` to align with the decision in the OZ update PR.

Reply:
Because this is based on the openzeppelin PR #401,  I will just rebase to the newer version.
Also because this is merging into the dev branch all the changes to the openzepplin are still marked here although they are not relevant to the PR